src/util/execute-lifecycle-script.js

```diff
@@ -184,18 +184,7 @@ export async function executeLifecycleScript(
 
   await checkForGypIfNeeded(config, cmd, env[constants.ENV_PATH_KEY].split(path.delimiter));
 
-  // get shell
-  let sh = 'sh';
-  let shFlag = '-c';
-
-  let windowsVerbatimArguments = undefined;
-
-  if (customShell) {
-    sh = customShell;
-  } else if (process.platform === 'win32') {
-    sh = process.env.comspec || 'cmd';
-    shFlag = '/d /s /c';
```

Comment:
Is it ok to remove these options? I have no idea what they do :/

`process.env.comspec` is also ignored with this PR

Reply:
Yes, the `shell: true` option takes care of all this. Please see #5133 which kind of undid what we already had here.

Reply:
Yeah but you also removed those options from the path that doesn't use `shell: true`, cf https://github.com/yarnpkg/yarn/pull/5484/files#diff-939cc0d319704d4f0d98baf9167ba850R212

Reply:
Nevermind I'm tired